railties/lib/rails/application.rb

```diff
@@ -111,8 +111,7 @@ def find_root(from)
       public :new
     end
 
-    attr_accessor :assets, :sandbox
-    alias_method :sandbox?, :sandbox
```

Comment:
I've used a new signature for sandbox method, to match the example from campfire `Rails.application.sandbox { ... }`.

Should we use a different name? A previous implementation of sandbox, that allows to run `Rails.application.sandbox = true` does not seem to be useful, taking in account that it could affect behavior only during a one-time check on startup.